packages/next/README.md

```diff
@@ -15,13 +15,46 @@
   </a>
 </p>
 
+## The React Framework
+
+Next.js is used by the world's leading companies and independent developers. 
+
+* **Pre-Rendering** - Statically generated and server-rendered React applications have never been easier.
+* **Static Exporting** - No need to learn a new framework. Exporting a static site with Next.js is as easy as a single command.
+* **CSS-in-JS** - Next.js comes with `styled-jsx` included, but it also works with every CSS-in-JS solution you know and love.
+* **Built-in TypeScript support** - Next.js provides an integrated TypeScript experience out of the box, similar to an IDE.
+* **Zero Configuration** - Automatic code splitting, filesystem based routing, hot code reloading and universal rendering. All included.
+* **Fully Extensible** - Complete control over Babel and Webpack. Customizable server, routing and next-plugins.
+* **Ready for Production** - Optimized for a smaller build size, faster dev compilation and dozens of other improvements.
+
```

Comment:
This is what the first page of our learning guide goes into, imo we should link to `/learn` asap instead of putting the same info here, feel free to also improve the learning page (https://nextjs.org/learn/basics/create-nextjs-app).

Reply:
I like the simple Next.js readme right now, but I feel like it would be useful to have this information on the repository as well. Happy to take this out if you think it's unnecessary.

Reply:
I don't think it's necessary, this is a redundant version of the first page in /learn. Wdyt @timneutkens ?

Reply:
@MehediH I still think we should remove this 👍